### PR TITLE
Add symphonik strategy

### PR DESCRIPTION
**Pull request by martimalek**
Add symphonik strategy

### CODE REVIEW

File: dry-run/symphonik/SymphonIK.py [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    # back into the trend at the next available opportunity, unless the trend has ended,
+    # in which case it would sell anyway.
+
+    # Stoploss:
+    stoploss = -0.10
+
+    def informative_pairs(self):
+        pairs = self.dp.current_whitelist()
+        informative_pairs = [(pair, self.informative_timeframe)
+                             for pair in pairs]
```
Comment (jorgealemangonzalez) [inline dry-run/symphonik/SymphonIK.py]: Delete this, as informative_timeframe = timeframe

File: dry-run/symphonik/SymphonIK.py [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+        if self.dp:
+            informative_pairs += [(pair, "3m") for pair in pairs]
+            informative_pairs += [(pair, "5m") for pair in pairs]
+            informative_pairs += [(pair, "10m") for pair in pairs]
```
Comment (jorgealemangonzalez) [inline dry-run/symphonik/SymphonIK.py]: 12h candle missing

Comment (jorgealemangonzalez) [reply dry-run/symphonik/SymphonIK.py]: BTC/USDT 4h missing. Check UP_DOWN strategy to see how to add it.

File: dry-run/symphonik/SymphonIK.py [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+            dataframe, dataframe4h, self.timeframe, "4h", ffill=True)
+
+        # Pares en 12h
+        dataframe12h = self.dp.get_pair_dataframe(
+            pair=metadata['pair'], timeframe="12h")
+
+        # MACD
+        macd = ta.MACD(dataframe12h)
+        dataframe['macd'] = macd['macd']
+        dataframe['macdsignal'] = macd['macdsignal']
```
Comment (jorgealemangonzalez) [inline dry-run/symphonik/SymphonIK.py]: merge_informative_pair missing

File: dry-run/symphonik/SymphonIK.py [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+                            )
+    dataframe[f'tenkan_sen_{conversion_line_period}'] = ichimoku['tenkan_sen']
+    dataframe[f'kijun_sen_{conversion_line_period}'] = ichimoku['kijun_sen']
+    dataframe[f'senkou_a_{conversion_line_period}'] = ichimoku['senkou_span_a']
+    dataframe[f'senkou_b_{conversion_line_period}'] = ichimoku['senkou_span_b']
+
+
+class SymphonIK(IStrategy):
+    # Optimal timeframe for the strategy
+    timeframe = '1m'
```
Comment (jorgealemangonzalez) [inline dry-run/symphonik/SymphonIK.py:92]: Do we want to act every minute or act every 12h ?

File: dry-run/symphonik/config.json [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+        "ccxt_config": {"enableRateLimit": true},
+        "ccxt_async_config": {
+            "enableRateLimit": true,
+            "rateLimit": 200
+        },
+        "pair_whitelist": [
+        ],
+        "pair_blacklist": [
+            "BNB/.*"
+        ]
```
Comment (jorgealemangonzalez) [inline dry-run/symphonik/config.json]: Add all laveraged coins ( UP, DOWN, BULL ... )

**Review by jorgealemangonzalez — CHANGES_REQUESTED** [review]
Please backtest first before merging.

File: dry-run/symphonik/SymphonIK.py [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+            pair=metadata['pair'], timeframe="10m")
+
+        dataframe10m['hma888'] = ftt.hull_moving_average(dataframe10m, 888)
+
+        dataframe = merge_informative_pair(
+            dataframe, dataframe10m, self.timeframe, "10m", ffill=True)
+
+        # BTC/USDT 4h
+
+        dataframe4h = self.dp.get_pair_dataframe(
```
Comment (jorgealemangonzalez) [inline dry-run/symphonik/SymphonIK.py]: I think this is not used any more

File: dry-run/symphonik/SymphonIK.py (hunk 1 above)
Comment (martimalek) [reply dry-run/symphonik/SymphonIK.py]: Do you mean that this line should be swapped by what you said? Isn't informative_pairs needed?

File: dry-run/symphonik/SymphonIK.py (hunk 4 above)
Comment (martimalek) [reply dry-run/symphonik/SymphonIK.py:92]: Every 1m

File: dry-run/symphonik/SymphonIK.py (hunk 1 above)
Comment (jorgealemangonzalez) [reply dry-run/symphonik/SymphonIK.py]: We can remove this line and the informative_pairs variable I think

Comment (martimalek) [reply dry-run/symphonik/SymphonIK.py]: I believe we can remove the whole function because it's not being called anywhere

**Review by jorgealemangonzalez — APPROVED** [review]

**Pull request merged**